### ISSUE TITLE
Fix @Code in Javadocs

### ISSUE THREAD

**Issue by eeverman** [eeverman/andhow#545]
A lot of the Javadocs use this notation:
`{@Code SomeChunkOfCode}`
That is great.... except 'Code' needs to be lowercase.

This is just a simple replacement, but if there are other Javadoc warnings that can be addressed as well - all the better.

**Comment by alex-kar** [comment]
I would like to work on it.
What I found so far:
```
@Code,@CODE => @code
@See        => @see
@depricated => @deprecated
```
But other than that there is an issue with `@` character inside `<pre>{@code}` tag.
For example
```
/**
	 * The default list of standard loaders, as a list of Classes that implement
	 * {@code StandardLoader}
	 * <p>
	 * The returned list is disconnected from the actual list of loaders - it is
	 * intended to be a starting point for applications that want to modify the
	 * list, then call setStandardLoaders().
	 * <p>
	 * Unlike other methods of this class, it does not fluently return a method
	 * to itself, so your code will need a AndHowConfiguration instance reference to
	 * use it, eg:
	 * <pre>{@code
	 * public class MyAppInitiation implements AndHowInit {
	 *    @Override
	 *  	public AndHowConfiguration getConfiguration() {
	 * 			AndHowConfiguration config = AndHow.findConfig();
	 * 			List<Class<? extends StandardLoader>> sll = config.getDefaultLoaderList();
	 * 			...do some rearranging of the list...
	 *
	 * 			config.setStandardLoaders(sll) ...and go on to call other methods on config...
	 * 		}
	 * }
	 * }</pre>
	 * <p>
	 * Note:  AndHow version up to and including 0.4.1 had this method as a static
	 * method.
	 * @return
	 */
```
generated to
```
List<Class<? extends StandardLoader>> getDefaultLoaderList()
The default list of standard loaders, as a list of Classes that implement StandardLoader
The returned list is disconnected from the actual list of loaders - it is intended to be a starting point for applications that want to modify the list, then call setStandardLoaders().

Unlike other methods of this class, it does not fluently return a method to itself, so your code will need a AndHowConfiguration instance reference to use it, eg:

{@code
 public class MyAppInitiation implements AndHowInit {
Returns:
```
I didn't find a solution to escape @ inside `<pre>{@code}`
(tried `@literal @`, `{@literal @}`, `&#064;`)

@eeverman WDYT?

**Comment by eeverman** [comment]
Its sad that that doesn't just work, but it looks like its an an open issue with the javadoc tool that can't be fixed.  The real fix will likely be a whole new tag or syntax to handle this better.

But until then... I was able to get a reasonable compromise like this:
```
	 * <p><pre>{@code
	 * public class MyAppInitiation implements AndHowInit {
	 *    //@Override //Commented out for Javadoc generation
	 *  	public AndHowConfiguration getConfiguration() {
	 * 			AndHowConfiguration config = AndHow.findConfig();
	 * 			AndHowConfiguration config = AndHow.findConfig();
	 * 			List<Class<? extends StandardLoader>> sll = config.getDefaultLoaderList();
	 * 			...do some rearranging of the list...
	 *
	 * 			config.setStandardLoaders(sll) ...and go on to call other methods on config...
	 * 		}
	 * }
	 * }</pre>
```
Putting something other than whitespace in front of the `@Override` fixes the issue:
`//@Override //Commented out for Javadoc generation`
The only thing that can be put there without making the code non-functional if it is copy/pasted is a comment.  The extra comment would just let a user know why it was commented in the first place.

Does that seem like a good solution?

**Comment by alex-kar** [comment]
I also tried to work around and found some solutions
Instead of use `<pre>{@code}` we can use html `<pre><code>`, but have to replace `@, <, >` with `&#64;`, `&lt;`, `&gt;`
Or use `{@code}` inside `<pre><code>`.
Few examples:
with `{@literal}` and `{@code}`
```
	 * <pre>
	 *     <code>
	 * public class MyAppInitiation implements AndHowInit {
	 *   	{@literal @}Override
	 *  	public AndHowConfiguration getConfiguration() {
	 * 			AndHowConfiguration config = AndHow.findConfig();
	 * 			{@code List<Class<? extends StandardLoader>> sll = config.getDefaultLoaderList();}
	 * 			...do some rearranging of the list...
	 *
	 * 			config.setStandardLoaders(sll) ...and go on to call other methods on config...
	 * 		}
	 * }
	 * </pre>
```
or escape special characters
```
	 * <pre>
	 *     <code>
	 * public class MyAppInitiation implements AndHowInit {
	 *   	&#64;Override
	 *  	public AndHowConfiguration getConfiguration() {
	 * 			AndHowConfiguration config = AndHow.findConfig();
	 * 			List&lt;Class&lt;? extends StandardLoader&gt;&gt; sll = config.getDefaultLoaderList();
	 * 			...do some rearranging of the list...
	 *
	 * 			config.setStandardLoaders(sll) ...and go on to call other methods on config...
	 * 		}
	 * }
	 * </pre>
```
both generate the same result
```
public class MyAppInitiation implements AndHowInit {
        @Override
        public AndHowConfiguration getConfiguration() {
                        AndHowConfiguration config = AndHow.findConfig();
                        List<Class<? extends StandardLoader>> sll = config.getDefaultLoaderList();
                        ...do some rearranging of the list...

                        config.setStandardLoaders(sll) ...and go on to call other methods on config...
                }
 }
```
But it becomes more complicated.
@eeverman Which solution do you prefer?

**Comment by eeverman** [comment]
@alex-kar Thanks for those examples.  Based on that, lets use:
* `<pre>{@code ....multi-line code...}</pre>` where there are no annotations
* `<pre><code> ...multi-line code {@code codeWith<AngleBraces> } ...</code></pre>` (your first example) when the code does have annotations

I prefer that combination because it requires the least modification to copy/paste code examples.  Thanks for your research on this - I hadn't even realized that this issue existed!

**Comment by eeverman** [comment]
Much thanks to @alex-kar on this this.
Fixed by #558
